go/teams/implicit.go

```diff
@@ -63,6 +89,20 @@ func LookupImplicitTeamIDUntrusted(ctx context.Context, g *libkb.GlobalContext,
 }
 
 func loadImpteamFromServer(ctx context.Context, g *libkb.GlobalContext, displayName string, public bool) (imp implicitTeam, err error) {
+	cacheKey := implicitTeamCacheKey{displayName: displayName, public: public}
+	cacher := g.GetImplicitTeamCacher()
+	if cacher != nil {
```

Comment:
engine can't call `NewImplicitTeamCacheAndInstall` because of a circular dependency unfortunately but chat tests do set this cache up